src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java

```diff
@@ -78,7 +78,7 @@ private boolean isLineMatching(AuditEvent event) {
                 final int startLine = aLineRangeList.get(0);
                 final int endLine = aLineRangeList.get(1);
                 final int currentLine = event.getLine();
-                result = currentLine > startLine && currentLine <= endLine;
+                result = currentLine >= startLine && currentLine <= endLine;
```

Comment:
please provide patch, input, config on where you caught it. Bug always result in extension of test code base.

Reply:
@HuGanghui my initial comment - https://github.com/checkstyle/patch-filters/pull/28#discussion_r444174777

> result = currentLine >= startLine && currentLine <= endLine;

Here's your second attempt to apply what I suggested - https://github.com/checkstyle/patch-filters/pull/39/files

> result = currentLine > startLine && currentLine <= endLine;

Please give attention to each single symbol next time you see proposed changes. If you don't agree or don't understand then ask.

Reply:
@rdiachenko I got it, I will be careful.

Reply:
@romani boundary test is here https://github.com/checkstyle/patch-filters/blob/27fb2668fc595a8db86896adecd7a572814e0397/src/test/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterTest.java#L83-L92 
https://github.com/checkstyle/patch-filters/blob/27fb2668fc595a8db86896adecd7a572814e0397/src/test/resources/com/github/checkstyle/patchfilter/BoundaryTestPatchOne.txt#L1-L5

Reply:
sorry, it's my oversight, sometimes I forget to execute the unit tests, and in the future I will add an ALL Test Pass to each PR description to ensure that ALL tests Pass.

Reply:
> boundary test is here, this example is only one line changed, so the startLine and endLine is same, when we change isLineMatching to

if you change main code you change test code also, please provide test/input/.... to cover this case.

Reply:
ok